test/dynamo/test_sources.py

```diff
@@ -0,0 +1,28 @@
+# Owner(s): ["module: dynamo"]
```

Comment:
test_source.py for symmetry with the module name in question?

Reply:
I wanted to, but then I saw test_misc has MiscTests, test_functions has FunctionTests, so I kept it as is lol